packages/svelte/src/internal/client/dom/elements/bindings/shared.js

```diff
@@ -34,7 +57,7 @@ export function listen(target, events, handler, call_handler_immediately = true)
  * @param {() => void} [on_reset]
  */
 export function listen_to_event_and_reset_event(element, event, handler, on_reset = handler) {
-	element.addEventListener(event, handler);
+	element.addEventListener(event, () => without_reactive_context(handler));
```

Comment:
why do we need this both here and where `listen_to_event_and_reset_event` is called?

Reply:
Where do we do that?

Reply:
Fixed